system-tests/e2e-suite/src/test/scala/com/hotels/styx/support/configuration/StyxBackend.scala

```diff
@@ -33,6 +34,12 @@ trait ImplicitOriginConversions {
     host = "localhost",
     port = fakeServer.port())
 
+  implicit def mockOrigin2Origin(server: MockOriginServer): Origin = Origin(
```

Comment:
It constructs an `Origin` object from `MockOriginServer` instance. That is, a function from MockOrigin to Origin. Number 2 is often used in place of word "to" because they sound similar. Using it here for the sake of consistency. The other conversion routines in the class follow similar naming.